javascript/ql/src/javascript.qll

```diff
@@ -34,6 +33,7 @@ import semmle.javascript.GlobalAccessPaths
 import semmle.javascript.HTML
 import semmle.javascript.HtmlSanitizers
 import semmle.javascript.InclusionTests
+import semmle.javascript.internal.CachedStages
```

Comment:
The point of being `internal` is sort of lost if we import it from `javascript.qll`.

Instead add private imports from the files that need it.